### PR TITLE
handle utf-8 and ignore case when searching dependency

### PR DESCRIPTION
**Pull request by mr-tz**
This should improve automatic updates for cyberchef (googlechrome vs. GoogleChrome) and other such cases.

### CODE REVIEW

**Comment by mr-tz** [comment]
Coincidentally, this could help with some update issues seen in https://github.com/mandiant/flare-vm/issues/437.

File: scripts/utils/update_package.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def update_github_url(package):
 # Metapackages have only one dependency and same name (adding `.vm`)  and version as the dependency
 def update_dependencies(package):
     nuspec_path = f"packages/{package}/{package}.nuspec"
-    with open(nuspec_path, "r") as file:
+    with open(nuspec_path, "r", encoding="utf-8") as file:
```
Comment (mr-tz) [inline scripts/utils/update_package.py:99]: see error: https://github.com/mandiant/VM-Packages/actions/runs/4052201794/jobs/6971349734#step:5:70

**Review by MalwareMechanic — APPROVED** [review]
Oh, good find!

**Comment by mr-tz** [comment]
Looks like it's working! https://github.com/mandiant/VM-Packages/pull/225

**Comment by tuttimann** [comment]
For me there are still many package update failures as seen in https://github.com/mandiant/flare-vm/issues/437

**Comment by mr-tz** [comment]
We haven't merged the proposed updates yet, so the packages are not available.
Hope @MalwareMechanic can shed some light on your referenced issue.

**Pull request merged**